src/lib/worldGenerations/grass_field.js

```diff
@@ -2,13 +2,16 @@ const Vec3 = require('vec3').Vec3
 
 function generation ({ version }) {
   const Chunk = require('prismarine-chunk')(version)
+  const mcData = require('minecraft-data')(version)
+  const theFlattening = require('../../../app').supportFeature('theFlattening')
```

Comment:
this important would start the server, that's not good.
Can you directly important the right thing?